Remove tests for custom RPM content

evgenyz opened the pull request:
We don't support or endorse this feature anymore.

evgenyz committed:
Remove tests for custom RPM content

We don't support or endorse this feature anymore.

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/OpenSCAP/oscap-anaconda-addon/inspections/73dd05c1-30e1-475e-92a1-48a547acc2ec) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/OpenSCAP/oscap-anaconda-addon/inspections/73dd05c1-30e1-475e-92a1-48a547acc2ec) completed: **No new issues**

evgenyz committed:
Do not rely on hashlib.algorithms_available

An algorithm could be a part of hashlib.algorithms_available set,
but still uninstantiatable in current runtime configuration.

See: https://github.com/python/cpython/issues/91257.

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/OpenSCAP/oscap-anaconda-addon/inspections/273d1f6a-18a0-4c98-b712-0e6fcc390aa9) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/OpenSCAP/oscap-anaconda-addon/inspections/273d1f6a-18a0-4c98-b712-0e6fcc390aa9) completed: **1 updated code elements**
